JavaScript/Task_Day1/Transportation_on_Vacation.js

```diff
@@ -0,0 +1,12 @@
+//https://www.codaysewars.com/kata/568d0dd208ee69389d000016/train/javascript
+
+
+// Transportation
+
+
+function rentalCarCost(days) {
+    var total=40*days;
+    if(days>=7)  total=total-50;
+    else if(days>=3) total=total-20;
+    return total;
```

Comment:
it is good to have `{ }` for `if-else` blocks as a practice even if they contain a single statement in them.